packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -868,9 +870,14 @@ export interface ISharedObjectKind<TSharedObject> {
  * @sealed
  * @public
  */
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface SharedObjectKind<out TSharedObject = unknown>
-	extends ErasedType<readonly ["SharedObjectKind", TSharedObject]> {}
+	extends ErasedType<readonly ["SharedObjectKind", TSharedObject]> {
+	/**
+	 * Check whether an {@link @fluidframework/core-interfaces#IFluidLoadable} is an instance of this shared object kind.
+	 * This should be used in place of `instanceof` checks for shared objects, as their actual classes are not exported in Fluid's public API.
```

Comment:
Nit (usage vs semantics):
```suggestion
	 * @remarks This should be used in place of `instanceof` checks for shared objects, as their actual classes are not exported in Fluid's public API.
```